datumaro/plugins/datumaro_format/extractor.py

```diff
@@ -98,30 +98,39 @@ def _load_items(self, parsed):
                     if osp.isfile(old_image_path):
                         image_path = old_image_path
 
-                image = Image(path=image_path, size=image_info.get('size'))
+                media = Image(path=image_path, size=image_info.get('size'))
+                self._media_type = Image
 
-            point_cloud = None
             pcd_info = item_desc.get('point_cloud')
-            if pcd_info:
+            if not media and pcd_info:
```

Comment:
I think, multiple media records should be considered an error.

Reply:
Added an error